Tests/conf.json

```diff
@@ -2803,6 +2803,11 @@
             "integrations": "Endace",
             "playbookID": "Endace-Test",
             "fromversion": "5.0.0"
+        },
+        {
+            "integrations": "Azure AD Connect Health Feed",
+            "playbookID": "FeedAzureADConnectHealth_Test",
+            "fromversion": "5.5.0"
```

Comment:
not needed, we do it automatically now.

Reply:
It fails pre-commit if I remove the entry in conf.json